pallets/external-validators/Cargo.toml

```diff
@@ -0,0 +1,77 @@
+[package]
+name = "pallet-external-validators"
+authors = { workspace = true }
+description = "Simple pallet to store external validators."
+edition = "2021"
+license = "GPL-3.0-only"
+version = "0.1.0"
+
+[package.metadata.docs.rs]
+targets = [ "x86_64-unknown-linux-gnu" ]
+
+[lints]
+workspace = true
+
+[dependencies]
+log = { workspace = true }
+parity-scale-codec = { workspace = true }
+rand = { workspace = true, optional = true }
+scale-info = { workspace = true, features = [ "derive" ] }
+
+frame-support = { workspace = true }
+frame-system = { workspace = true }
+sp-runtime = { workspace = true }
+sp-staking = { workspace = true }
+sp-std = { workspace = true }
+tp-traits = { workspace = true }
+
+frame-benchmarking = { workspace = true }
+
+pallet-balances = { workspace = true, optional = true }
```

Comment:
do we need pallet-balances at all?

Reply:
maybe I am wrong but it looks to me you only need it for benches

Reply:
It's already optional, only enabled in benchmarks

Reply:
*disregard*